zebedee-cms/src/main/java/com/github/onsdigital/zebedee/persistence/model/CollectionEventMetaData.java

```diff
@@ -91,10 +90,10 @@ public static CollectionEventMetaData create(String key, String value) {
      * Create a {@link CollectionEventMetaData} for viewer team removed event.
      */
     public static CollectionEventMetaData[] teamRemoved(CollectionDescription collectionDescription,
-                                                        Session session, Team team) throws IOException, ZebedeeException {
+                                                        Session session, Integer teamId) throws IOException, ZebedeeException {
```

Comment:
If you make this `int` instead of `Integer` it removes the need to do a null check.